jekyll/_cci2/github-apps-integration.adoc

```diff
@@ -0,0 +1,255 @@
+
+You can add or delete a configuration source for your project. If you followed the steps above to connect GitHub, a GitHub configuration source has been automatically added for you.
+
+Once you define a configuration source, you can set up a trigger that points to that configuration.
+
+image::{{site.baseurl}}/assets/img/docs/gl-ga/gitlab-ga-project-settings-configuration.png[Configuration setup page]
+
+[#triggers]
+=== Triggers
+
+**The scheduled pipelines feature is not currently available for use with GitHub App integrations.** Triggers are described below, including how to use filters to trigger pipelines based on certain conditions.
+
+Add a trigger that specifies which configuration source starts a pipeline. If you followed the steps above to connect GitHub, a trigger set with GitHub as the configuration source has been automatically added for you.
+
+image::{{site.baseurl}}/assets/img/docs/gl-ga/gitlab-ga-project-settings-triggers.png[Trigger setup page]
+
+Triggers and trigger rules determine how CircleCI handles events from the source of change, in this case, GitHub.
+
+When a trigger is created, CircleCI registers a webhook with GitLab. Push events from GitLab are sent to CircleCI. CircleCI then uses the event data to determine _if_ a pipeline should run, and if so, _which_ pipeline should be run.
+
+In addition to a configuration source, each trigger includes the webhook URL, and in this scenario, a CircleCI-created GitLab token. The webhook URL and GitLab token are used to securely register the webhook within GitLab in order to receive push events from your GitLab repo.
+
+image::{{site.baseurl}}/assets/img/docs/gl-ga/gitlab-ga-project-settings-edit-trigger.png[Trigger details]
+
+**Trigger filters** allow you to determine when a trigger should initiate a build based on the parameters provided by Gitlab’s webhook. CircleCI provides some common options, for example, only build on merge requests, but you can also build your own rules using the custom filter option. For example, a custom filter would allow you to only build on a specific branch or user.
+
+image::{{site.baseurl}}/assets/img/docs/gl-preview/gitlab-preview-project-settings-customize-triggers.png[Trigger details]
+
+NOTE: Currently, the only fields that can be edited for existing filters are **Trigger Name** and the **Filters** radio buttons.
+
+[#project-settings-advanced]
+=== Advanced
+
+- You can enable dynamic configuration using setup workflows in CircleCI. To learn about dynamic configuration, read the xref:dynamic-config#[Dynamic configuration] guide.
+- At this time, the **Free and Open Source** setting is not currently supported, but there are plans to make this available in the future.
+- At this time, auto-cancel redundant workflows is not supported. Refer to the xref:skip-build#auto-cancelling[Auto cancelling] section of the `skip` or `cancel` jobs and workflows page for more details.
+
+[#project-settings-ssh-keys]
+=== GitHub project SSH keys
+
```

Comment:
@Andrewwelton Also here, I'm not 100 sure on the flow for adding SSH keys for code checkout. I know users will be guided through this in the app but thought we should also describe the process here